worker/upload.go

```diff
@@ -336,19 +337,18 @@ func (w *worker) threadedUploadPackedSlabs(rs api.RedundancySettings, contractSe
 }
 
 func (w *worker) tryUploadPackedSlabs(ctx context.Context, rs api.RedundancySettings, contractSet string, block bool) (err error) {
-	// if we want to block, try and upload one packed slab synchronously
+	// if we want to block, try and upload one packed slab synchronously, we use
+	// a slightly higher upload priority to avoid reaching the context deadline
 	if block {
-		ctx, cancel := context.WithTimeout(ctx, defaultPackedSlabsUploadTimeout)
-		defer cancel()
-		_, err = w.uploadPackedSlabs(ctx, defaultPackedSlabsLockDuration, rs, contractSet, defaultPackedSlabsLimit)
+		_, err = w.uploadPackedSlabs(ctx, defaultPackedSlabsLockDuration, rs, contractSet, defaultPackedSlabsLimit, lockingPriorityUpload+1)
 	}
 
 	// make sure there's a goroutine uploading the remainder of the packed slabs
-	go w.threadedUploadPackedSlabs(rs, contractSet)
+	go w.threadedUploadPackedSlabs(rs, contractSet, lockingPriorityUpload-1)
```

Comment:
I'd prefer an explicit const instead of `lockingPriorityUpload-1`. That way it's all in one place and it's easier to see what kind of priorities we use throughout the application.